packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -1665,6 +1688,18 @@ class ImportedCluster extends ClusterBase implements ICluster {
     }
     return this.props.clusterEncryptionConfigKeyArn;
   }
+
+  /**
+   * If OpenID Connect issuer url was configured by clusterAttributes while import,
+   * this will return a value. Otherwise this will be `undefined`
+   * @attribute
+   */
+  public get clusterOpenIdConnectIssuerUrl(): string {
```

Comment:
No need after you remove the `clusterOpenIdConnectIssuerUrl` from the `ICluster` interface.

Reply:
see my comment to the ICluster